ts/src/flexible-event/main.ts

```diff
@@ -90,6 +103,8 @@ if (options.json_file !== undefined) {
     (source) =>
       new Config(
         source.maxEventLevelReports,
+        source.attributionScopeLimit ?? 1,
+        source.maxEventStates ?? constants.defaultMaxEventStates,
```

Comment:
The righthand side of the `??` will never be taken, as `maxEventStates` is guaranteed not to be `null` or `undefined`.

Reply:
This hasn't been resolved.